[07/14-05:15:15] dbtestmon: bind() to controlDBIp failed 
                 errno: Address already in use
